### ISSUE TITLE
How do I create a new TEI XML file in Oxygen?

### ISSUE THREAD

**Issue by gabrielbodard** [SASDigitalHumanitiesTraining/TextEncoding#7]

**Comment by gabrielbodard** [comment]
I'm glad you asked. If you're using Oxygen, as we suggested, there is a demo of this at the beginning of the [Tagging Text Structure](https://youtu.be/zkL_JJFBSLU) tutorial video, but for reference, the process is as follows:

1. From "File" menu, select "New…"
2. From "Select or search a document type" select "Framework templates" --> "TEI P5" --> "All [TEI P5]"
3. Save this file to your computer, and replace the contents of `<body>` with your text (or `<sourceDesc>` with your descriptive and historical data).

If you're not using Oxygen, you can also find a copy of this blank TEI XML file [at this link](https://github.com/SASDigitalHumanitiesTraining/TextEncoding/raw/master/blank-tei-document.xml) (right-click and "Save link as…" before opening the file in your XML editor).

**Comment by cbeattie19** [comment]
Under framework templates, I only see TEI and not TEI P5. Can I use that?
[Screen Shot 2021-05-17 at 16 21 56](https://user-images.githubusercontent.com/84335537/118514060-24df7380-b72c-11eb-87c8-952d4ab57073.png)

**Comment by strill66** [comment]
@cbeattie19 that's what I see also

**Comment by cmohge1** [comment]
That file template should work. As long as you see the following below, then you'll be OK.

`<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://www.tei-c.org/release/xml/tei/custom/schema/relaxng/tei_all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://www.tei-c.org/release/xml/tei/custom/schema/relaxng/tei_all.rng" type="application/xml"
	schematypens="http://purl.oclc.org/dsdl/schematron"?>
<TEI xmlns="http://www.tei-c.org/ns/1.0">
  <teiHeader>
      <fileDesc>
         <titleStmt>
            <title>Title</title>
         </titleStmt>
         <publicationStmt>
            <p>Publication Information</p>
         </publicationStmt>
         <sourceDesc>
            <p>Information about the source</p>
         </sourceDesc>
      </fileDesc>
  </teiHeader>
  <text>
      <body>
         <p>Some text here.</p>
      </body>
  </text>
</TEI>`

**Comment by NicolePohl** [comment]
When I click on TEI, the above choices come up so which do I choose?  I am afraid that the videos are really not clear to use Oxygen - I do not understand at all what to do. I downloaded Oxygen, but how do I use a manuscript text and copy it into Oxygen to try to encode it?

**Comment by strill66** [comment]
@NicolePohl  - in my experience it should be possible to copy a transcription from a word file, paste it into the <body><p> section of the Oxygen file, and work from there

**Comment by cmohge1** [comment]
Thanks @strill66 . @NicolePohl indeed you should copy and paste a text into the `<body>`. We have provided text examples you could copy and paste from the Exercises section under **Introduction to TEI** –– e.g. you could use the text at https://www.perseus.tufts.edu/hopper/text?doc=Perseus%3Atext%3A1999.01.0217%3Abook%3D1%3Acard%3D1

```
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://www.tei-c.org/release/xml/tei/custom/schema/relaxng/tei_all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://www.tei-c.org/release/xml/tei/custom/schema/relaxng/tei_all.rng" type="application/xml"
	schematypens="http://purl.oclc.org/dsdl/schematron"?>
<TEI xmlns="http://www.tei-c.org/ns/1.0">
  <teiHeader>
      <fileDesc>
         <titleStmt>
            <title>Title</title>
         </titleStmt>
         <publicationStmt>
            <p>Publication Information</p>
         </publicationStmt>
         <sourceDesc>
            <p>Information about the source</p>
         </sourceDesc>
      </fileDesc>
  </teiHeader>
  <text>
      <body>
         <p>Sing, O goddess, the anger [mênis] of Achilles son of Peleus, that brought countless ills upon the Achaeans. Many a brave soul [psukhê] did it send hurrying down to Hades, and many a hero did it yield a prey to dogs</p>
      </body>
  </text>
</TEI>
```
You'll of course need to make more changes, but this should get you started. Does that help?

**Comment by gabrielbodard** [comment]
Yes, apologies, I have a slightly older version of Oxygen, so the menus are a little different. Revising my instructions from above:

1. From "File" menu, select "New…"
1. From "Select or search a document type" select "Framework templates" --> "TEI" --> "All"
1. Save this file to your computer, and open with Oxygen or any text editor.
1. In the open file, replace the contents of `<body>` with your text (as Christopher showed in his example above).
